Frameworks/Foundation/NSUserDefaults.mm

```diff
@@ -119,16 +132,44 @@ - (void)setPersistentDomain:(id)domain forName:(NSString*)name {
  @Status Interoperable
```

Comment:
>Interoperable [](start = 9, length = 13)

this should really be caveat - with a note that synchronize does not read from file